keyboards/basekeys/slice/keymaps/default/keymap.c

```diff
@@ -0,0 +1,161 @@
+#include QMK_KEYBOARD_H
+
```

Comment:
```suggestion
#include "split_util.h"

```

Reply:
3 Keymaps were updated with this change.

keymaps/default/keymap.c
keymaps/default_split_left_space/keymap.c
rev1_rgb/keymaps/slice_rgb/keymap.c

Reply:
3 Keymaps were updated with this change.

keymaps/default/keymap.c
keymaps/default_split_left_space/keymap.c
rev1_rgb/keymaps/slice_rgb/keymap.c